broker-daemon/block-order-worker/index.js

```diff
@@ -566,15 +566,27 @@ class BlockOrderWorker extends EventEmitter {
    * @returns {void}
    */
   async workLimitBlockOrder (blockOrder, targetDepth) {
-    if (blockOrder.timeInForce !== BlockOrder.TIME_RESTRICTIONS.GTC) {
-      throw new Error('Only Good-til-cancelled limit orders are currently supported.')
+    if ((blockOrder.timeInForce !== BlockOrder.TIME_RESTRICTIONS.GTC) && (blockOrder.timeInForce !== BlockOrder.TIME_RESTRICTIONS.PO)) {
+      throw new Error('Only Good-til-cancelled and Post Only limit orders are currently supported.')
```

Comment:
do you still need this with the other error handler? Seems like it can be removed.

Reply:
Agreed, I'll take it out.